teamcity: failed tests on master: testrace/Example_cclzone, test/Example_cclzone

The following tests appear to have failed:

[#407618](https://teamcity.cockroachdb.com/viewLog.html?buildId=407618):


```
--- FAIL: testrace/Example_cclzone (0.720s)

------- Stdout: -------
Error: pq: syntax error at or near "("
DETAIL: source SQL:
CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
                      ^
HINT: try \h CREATE TABLE
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t@nonexistent" does not exist
Error: pq: relation "db.t" does not exist
Error: index and partition cannot be specified simultaneously: "db.t.p0@t_c2_idx"
Error: pq: relation "db.t@primary" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t@t_c2_idx" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t@primary" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
W171113 16:20:41.417047 146 vendor/google.golang.org/grpc/transport/log.go:36  transport: http2Server.HandleStreams failed to read frame: io: read/write on closed pipe
W171113 16:20:41.417172 280 vendor/google.golang.org/grpc/clientconn.go:934  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: Error while dialing dial tcp 127.162.134.157:41264: operation was canceled"; Reconnecting to {127.162.134.157:41264 0  <nil>}
W171113 16:20:41.417366 280 vendor/google.golang.org/grpc/clientconn.go:1028  grpc: addrConn.transportMonitor exits due to: context canceled
W171113 16:20:41.417460 134 vendor/google.golang.org/grpc/clientconn.go:934  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: failed to write client preface: io: read/write on closed pipe"; Reconnecting to {127.162.134.157:41264 0  <nil>}
W171113 16:20:41.417553 134 vendor/google.golang.org/grpc/clientconn.go:1028  grpc: addrConn.transportMonitor exits due to: context canceled
got:
sql -e CREATE DATABASE db
CREATE DATABASE
sql -e CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
  PARTITION p1 VALUES (DEFAULT)
)
pq: syntax error at or near "("
DETAIL: source SQL:
CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
                      ^
HINT: try \h CREATE TABLE
sql -e CREATE INDEX ON db.t (c2)
pq: relation "db.t" does not exist
zone set db.t@nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: relation "db.t@nonexistent" does not exist
zone set db.t.nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: relation "db.t" does not exist
zone set db.t.p0@t_c2_idx --file=./../../cli/testdata/zone_attrs.yaml
index and partition cannot be specified simultaneously: "db.t.p0@t_c2_idx"
zone set db.t@primary --file=./../../cli/testdata/zone_attrs.yaml
pq: relation "db.t@primary" does not exist
zone get db.t.p0
pq: relation "db.t" does not exist
zone get db.t
pq: relation "db.t" does not exist
zone get db.t@t_c2_idx
pq: relation "db.t@t_c2_idx" does not exist
zone set db.t.p1 --file=./../../cli/testdata/zone_range_max_bytes.yaml
pq: relation "db.t" does not exist
zone get db.t.p1
pq: relation "db.t" does not exist
zone get db.t.p0
pq: relation "db.t" does not exist
zone ls
.default
zone rm db.t@primary
pq: relation "db.t@primary" does not exist
zone get db.t.p0
pq: relation "db.t" does not exist
zone get db.t.p1
pq: relation "db.t" does not exist
zone ls
.default
zone rm db.t.p0
pq: relation "db.t" does not exist
zone rm db.t.p1
pq: relation "db.t" does not exist
zone ls
.default
want:
sql -e CREATE DATABASE db
CREATE DATABASE
sql -e CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
  PARTITION p1 VALUES (DEFAULT)
)
CREATE TABLE
sql -e CREATE INDEX ON db.t (c2)
CREATE INDEX
zone set db.t@nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: index "nonexistent" does not exist
zone set db.t.nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: partition "nonexistent" does not exist
zone set db.t.p0@t_c2_idx --file=./../../cli/testdata/zone_attrs.yaml
index and partition cannot be specified simultaneously: "db.t.p0@t_c2_idx"
zone set db.t@primary --file=./../../cli/testdata/zone_attrs.yaml
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: [us-east-1a, ssd]
zone get db.t.p0
db.t@primary
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: [us-east-1a, ssd]
zone get db.t
.default
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: []
zone get db.t@t_c2_idx
.default
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: []
zone set db.t.p1 --file=./../../cli/testdata/zone_range_max_bytes.yaml
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 90000
num_replicas: 3
constraints: [us-east-1a, ssd]
zone get db.t.p1
db.t.p1
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 90000
num_replicas: 3
constraints: [us-east-1a, ssd]
zone get db.t.p0
db.t@primary
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: [us-east-1a, ssd]
zone ls
.default
db.t.p1
db.t@primary
zone rm db.t@primary
CONFIGURE ZONE 1
zone get db.t.p0
.default
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: []
zone get db.t.p1
db.t.p1
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 90000
num_replicas: 3
constraints: [us-east-1a, ssd]
zone ls
.default
db.t.p1
zone rm db.t.p0
CONFIGURE ZONE 0
zone rm db.t.p1
CONFIGURE ZONE 1
zone ls
.default
--- FAIL: test/Example_cclzone (0.150s)

------- Stdout: -------
Error: pq: syntax error at or near "("
DETAIL: source SQL:
CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
                      ^
HINT: try \h CREATE TABLE
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t@nonexistent" does not exist
Error: pq: relation "db.t" does not exist
Error: index and partition cannot be specified simultaneously: "db.t.p0@t_c2_idx"
Error: pq: relation "db.t@primary" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t@t_c2_idx" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t@primary" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
Error: pq: relation "db.t" does not exist
W171113 16:22:51.001743 237 vendor/google.golang.org/grpc/clientconn.go:934  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: Error while dialing dial tcp 127.152.83.20:33109: operation was canceled"; Reconnecting to {127.152.83.20:33109 0  <nil>}
W171113 16:22:51.001816 237 vendor/google.golang.org/grpc/clientconn.go:1028  grpc: addrConn.transportMonitor exits due to: context canceled
W171113 16:22:51.001839 130 vendor/google.golang.org/grpc/transport/log.go:36  transport: http2Server.HandleStreams failed to read frame: io: read/write on closed pipe
W171113 16:22:51.001856 259 vendor/google.golang.org/grpc/transport/log.go:36  transport: http2Server.HandleStreams failed to read frame: read tcp 127.152.83.20:33109->127.0.0.1:56820: use of closed network connection
W171113 16:22:51.001987 118 vendor/google.golang.org/grpc/clientconn.go:934  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: failed to write client preface: io: read/write on closed pipe"; Reconnecting to {127.152.83.20:33109 0  <nil>}
W171113 16:22:51.002025 118 vendor/google.golang.org/grpc/clientconn.go:1028  grpc: addrConn.transportMonitor exits due to: context canceled
got:
sql -e CREATE DATABASE db
CREATE DATABASE
sql -e CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
  PARTITION p1 VALUES (DEFAULT)
)
pq: syntax error at or near "("
DETAIL: source SQL:
CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
                      ^
HINT: try \h CREATE TABLE
sql -e CREATE INDEX ON db.t (c2)
pq: relation "db.t" does not exist
zone set db.t@nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: relation "db.t@nonexistent" does not exist
zone set db.t.nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: relation "db.t" does not exist
zone set db.t.p0@t_c2_idx --file=./../../cli/testdata/zone_attrs.yaml
index and partition cannot be specified simultaneously: "db.t.p0@t_c2_idx"
zone set db.t@primary --file=./../../cli/testdata/zone_attrs.yaml
pq: relation "db.t@primary" does not exist
zone get db.t.p0
pq: relation "db.t" does not exist
zone get db.t
pq: relation "db.t" does not exist
zone get db.t@t_c2_idx
pq: relation "db.t@t_c2_idx" does not exist
zone set db.t.p1 --file=./../../cli/testdata/zone_range_max_bytes.yaml
pq: relation "db.t" does not exist
zone get db.t.p1
pq: relation "db.t" does not exist
zone get db.t.p0
pq: relation "db.t" does not exist
zone ls
.default
zone rm db.t@primary
pq: relation "db.t@primary" does not exist
zone get db.t.p0
pq: relation "db.t" does not exist
zone get db.t.p1
pq: relation "db.t" does not exist
zone ls
.default
zone rm db.t.p0
pq: relation "db.t" does not exist
zone rm db.t.p1
pq: relation "db.t" does not exist
zone ls
.default
want:
sql -e CREATE DATABASE db
CREATE DATABASE
sql -e CREATE TABLE db.t (
  c1 STRING PRIMARY KEY,
  c2 STRING
) PARTITION BY LIST (c1) (
  PARTITION p0 VALUES ('a'),
  PARTITION p1 VALUES (DEFAULT)
)
CREATE TABLE
sql -e CREATE INDEX ON db.t (c2)
CREATE INDEX
zone set db.t@nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: index "nonexistent" does not exist
zone set db.t.nonexistent --file=./../../cli/testdata/zone_attrs.yaml
pq: partition "nonexistent" does not exist
zone set db.t.p0@t_c2_idx --file=./../../cli/testdata/zone_attrs.yaml
index and partition cannot be specified simultaneously: "db.t.p0@t_c2_idx"
zone set db.t@primary --file=./../../cli/testdata/zone_attrs.yaml
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: [us-east-1a, ssd]
zone get db.t.p0
db.t@primary
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: [us-east-1a, ssd]
zone get db.t
.default
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: []
zone get db.t@t_c2_idx
.default
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: []
zone set db.t.p1 --file=./../../cli/testdata/zone_range_max_bytes.yaml
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 90000
num_replicas: 3
constraints: [us-east-1a, ssd]
zone get db.t.p1
db.t.p1
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 90000
num_replicas: 3
constraints: [us-east-1a, ssd]
zone get db.t.p0
db.t@primary
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: [us-east-1a, ssd]
zone ls
.default
db.t.p1
db.t@primary
zone rm db.t@primary
CONFIGURE ZONE 1
zone get db.t.p0
.default
range_min_bytes: 1048576
range_max_bytes: 67108864
gc:
  ttlseconds: 90000
num_replicas: 1
constraints: []
zone get db.t.p1
db.t.p1
range_min_bytes: 1048576
range_max_bytes: 134217728
gc:
  ttlseconds: 90000
num_replicas: 3
constraints: [us-east-1a, ssd]
zone ls
.default
db.t.p1
zone rm db.t.p0
CONFIGURE ZONE 0
zone rm db.t.p1
CONFIGURE ZONE 1
zone ls
.default

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/ccl/cliccl/cli_test.go
+++ b/pkg/ccl/cliccl/cli_test.go
@@ -69,8 +69,8 @@ func Example_cclzone() {
   c1 STRING PRIMARY KEY,
   c2 STRING
 ) PARTITION BY LIST (c1) (
-  PARTITION p0 VALUES ('a'),
-  PARTITION p1 VALUES (DEFAULT)
+  PARTITION p0 VALUES IN ('a'),
+  PARTITION p1 VALUES IN (DEFAULT)
 )`})
 	c.runWithArgs([]string{"sql", "-e", "CREATE INDEX ON db.t (c2)"})
 	c.run("zone set db.t@nonexistent --file=./../../cli/testdata/zone_attrs.yaml")
@@ -99,8 +99,8 @@ func Example_cclzone() {
 	//   c1 STRING PRIMARY KEY,
 	//   c2 STRING
 	// ) PARTITION BY LIST (c1) (
-	//   PARTITION p0 VALUES ('a'),
-	//   PARTITION p1 VALUES (DEFAULT)
+	//   PARTITION p0 VALUES IN ('a'),
+	//   PARTITION p1 VALUES IN (DEFAULT)
 	// )
 	// CREATE TABLE
 	// sql -e CREATE INDEX ON db.t (c2)

